packages/tasks/src/library-ui-elements.ts

```diff
@@ -774,6 +782,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS: Partial<Record<ModelLibraryKey, Librar
 		docsUrl: "https://huggingface.co/docs/hub/ml-agents",
 		snippets: mlAgents,
 	},
+	"unity-sentis": {
+		btnLabel: "unity sentis",
```

Comment:
I'd maybe use `unity-sentis` for consistency with the tag. This is how it's displayed in the case of `ml-agents`:

<img width="437" alt="Screenshot 2024-01-17 at 20 11 00" src="https://github.com/huggingface/huggingface.js/assets/1177582/5094d848-a543-4f0a-8c3a-42298ed5d1e1">